modules/rostests/apitests/ntdll/NtDuplicateToken.c

```diff
@@ -80,15 +92,11 @@ DuplicateTokenAsEffective(VOID)
 
     /* Get the token from process and begin the tests */
     TokenHandle = OpenTokenFromProcess();
-
-    /* We give a bogus invalid handle */
-    Status = NtDuplicateToken(NULL,
-                              0,
-                              NULL,
-                              TRUE,
-                              TokenPrimary,
-                              NULL);
-    ok_hex(Status, STATUS_ACCESS_VIOLATION);
+    if (TokenHandle == NULL)
+    {
+        skip("TokenHandle is NULL\n");
+        return;
+    }
```

Comment:
This is pointless because this is a situation that's highly unlikely to happen for the current calling thread of the process to not have the token.